SU2_CFD/src/iteration_structure.cpp

```diff
@@ -2127,6 +2127,7 @@ void CDiscAdjFluidIteration::Preprocess(COutput *output,
   unsigned short iMesh;
   int Direct_Iter;
   bool heat = config[val_iZone]->GetWeakly_Coupled_Heat();
+  bool grid_movement_bool = config[val_iZone]->GetGrid_Movement();
```

Comment:
Can you please switch 'grid_movement_bool' to 'grid_movement' to keep with the convention throughout the rest of the code? Eventually, we could use something more expressive for the name like 'gridIsMoving' (find and replace everywhere), but for now, consistency is enough

Reply:
Yep, but I would actually prefer `bool gridIsMoving` as `grid_movement `is used as a larger container for `CVolumetricMovement ***grid_movement`.